pkg/artifactory/resource/repository/remote/resource_artifactory_remote_repository_test.go

```diff
@@ -1211,7 +1210,7 @@ func TestAccRemoteRepository_generic_migrate_to_schema_v4(t *testing.T) {
 				ExternalProviders: map[string]resource.ExternalProvider{
 					"artifactory": {
 						Source:            "jfrog/artifactory",
-						VersionConstraint: "12.0.0",
+						VersionConstraint: "12.3.3",
```

Comment:
This is a TF schema migration test so the version constraint is specific to `12.0.0` (the previous release version with schema v3). See PR: https://github.com/jfrog/terraform-provider-artifactory/pull/1085/files